scripts/push-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+file="$1"
+
+prefix="$(cat $file | jq -r '.package .code')"
+version="$(cat $file | jq -r '.package .DHIS2Version' | cut -d '.' -f 1,2)"
+
+package_code="${prefix:0:4}"
+sub_package_code="${prefix:4:2}"
+
+repo=$(
+  curl -fsSL "https://api.github.com/orgs/dhis2-metadata/repos?per_page=100" |
+  jq -r --arg package_code "$package_code" '.[] | select(.name | contains($package_code)) | .name'
+)
+
+git config --global user.email "$GITHUB_EMAIL"
+git config --global user.name "$GITHUB_CREDS_USR"
```

Comment:
i can't find any references to GITHUB_CREDS_USR, but I'm assuming it's a secret and won't be printed?